mouse/record.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+
+
+import argparse
+import io
+import time
+import picamera
+import datetime
+import os
+
+
+
+def main(*argv):
+   """
+       Defines arguments: time before and after, and mouse/trial to be tested
+
+   """
+   
+   
+   parser = argparse.ArgumentParser(description='seconds before and after lever is pressed.')
```

Comment:
Indentation throughout file is inconsistent. Indent should multiples of 4 spaces. PyCharm can help you do this ( https://www.jetbrains.com/pycharm/help/reformatting-source-code.html ).